tachyon/zk/expressions/expression_unittest.cc

```diff
@@ -52,26 +39,23 @@ TEST_F(ExpressionTest, ArithmeticOperatorWithClone) {
   std::unique_ptr<Expression<F>> right =
       base::UniformElement(expressions_)->Clone();
 
-  if (!ContainsSimpleSelector(left.get()) &&
-      !ContainsSimpleSelector(right.get())) {
-    std::unique_ptr<Expression<F>> add = left + right;
-    EXPECT_EQ(*add->ToSum()->left(), *left);
-    EXPECT_EQ(*add->ToSum()->right(), *right);
-    EXPECT_TRUE(left);
-    EXPECT_TRUE(right);
```

Comment:
It looks like these tests are gone from this commit. Could you add this test to somethign like `tachyon/zk/plonk/expressions_unittest.cc`?